bolt-design-system/bolt
# Bolt table to web component

## Jira

http://vjira2:8080/browse/BDS-658

## Summary

Conversion bolt table component to web component

## Details

Add web component support to bolt table.

## How to test

Run this code locally and check if all bolt table examples are generated correctly with js and without.
Check web component examples on the bottom of bolt table component page.
